source/_integrations/alarm_control_panel.template.markdown

```diff
@@ -108,7 +108,3 @@ panels:
 If you are using the state of a integration that takes extra time to load, the Template Alarm Control Panel may get an `unknown` state during startup. This results in error messages in your log file until that integration has completed loading. If you use `is_state()` function in your template, you can avoid this situation.
 
 For example, you would replace {% raw %}`{{ states.switch.source.state == 'on' }}`{% endraw %} with this equivalent that returns `true`/`false` and never gives an unknown result: {% raw %}`{{ is_state('switch.source', 'on') }}`{% endraw %}
-
-### Working without entities
```

Comment:
These sections still apply, right?

`{{ range(0,100)|random }}`

Reply:
The `random` does not cause automatic updates but I don't think it's worth having these sections just for that, it would cause confusion rather than clear it up IMO.

Reply:
Honestly, I do not agree, especially since I'm not making the example above up. It was a genuine question I had from people.

I don't see how that example is any different from `now()`. Just because `now()` is handled, doesn't render these sections invalid.

Reply:
Okay but the text removed here never did much to address `random`. I added a new subsection about this question in #15394.